wrc/codegen/cghtml.py

```diff
@@ -76,8 +76,10 @@ def link2html(text):
 def simple_md2html(text, urls):
     ''' Convert a text from md to html '''
     retval = special_links_replace(text, urls)
-    # Create a br for every 4 spaces
-    retval = re.sub(r'[ ]{4}\n', r'<br />\n', retval)
+    # Create a par break for double newlines
+    retval = re.sub(r'\n\n', r'</p><p>', retval)
```

Comment:
This looks like a recipe for unbalanced `<p>` tags. How do we ensure that the generated HTML has a closing `</p>` for every opening `<p>`?